pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -237,6 +237,7 @@ func (o *MustGatherOptions) Run() error {
 			},
 			Annotations: map[string]string{
 				"oc.openshift.io/command": "oc adm must-gather",
+				"openshift.io/node-selector": "",
```

Comment:
yeah, this seems fine.  YOu want it for oc debug node, too, right?

Reply:
yes, i added that in a [separate PR](https://github.com/openshift/oc/pull/550), bc wasn't sure on direction of that.